roles/lib_openshift/library/oc_adm_ca_server_cert.py

```diff
@@ -1080,7 +1080,7 @@ def _run(self, cmds, input_data):
 
         stdout, stderr = proc.communicate(input_data)
 
-        return proc.returncode, stdout.decode(), stderr.decode()
+        return proc.returncode, stdout.decode('utf-8'), stderr.decode('utf-8')
```

Comment:
## 2
### str
Becomes ``unicode``
### unicode
Stays ``unicode``

## 3
### bytes
Become ``str``
### str (which shouldn't be returned in stdout/stderr)
Will error.